var/spack/repos/builtin/packages/py-salib/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySalib(PythonPackage):
+    """Python implementations of commonly used sensitivity analysis methods."""
+
+    homepage = "http://salib.readthedocs.org"
+    git      = "https://github.com/SALib/SALib"
+
+    maintainers = ['schmitts']
+
+    version('1.3.13', tag='v1.3.13')
+
+    depends_on('py-pip', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-wheel', type=('build', 'run'))
+
+    depends_on('py-numpy@1.9.0:', type=('run'))
+    depends_on('py-scipy', type=('run'))
+    depends_on('py-matplotlib@1.4.3:', type=('run'))
+    depends_on('py-pandas', type=('run'))
```

Comment:
These should all be `type=('build', 'run')`

Reply:
Where did you find the version requirements?

Reply:
https://github.com/SALib/SALib/blob/master/requirements.txt

Reply:
> These should all be `type=('build', 'run')`

How  (except try and error) could I determine if the dep. is needed during build?

Reply:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies

Reply:
I wouldn't add the version restrictions from master since these aren't present in the release you are adding.